src/modules/admin/components/formRules/FormRuleTable.tsx

```diff
@@ -73,17 +57,22 @@ const FormRuleTable: React.FC<Props> = ({ formId, formRole }) => {
             <TableCell sx={{ py: 1 }}>
               <FormRule
                 rule={rule}
-                setSelectedRule={setSelectedRule}
-                openFormDialog={openFormDialog}
+                formRole={formRole}
+                actionButtons={
+                  <IconButton
+                    onClick={() => deactivate({ variables: { id: rule.id } })}
+                    size='small'
+                    sx={{ height: '28px' }}
+                    disabled={loading}
+                  >
+                    <DeleteIcon fontSize='small' />
+                  </IconButton>
+                }
```

Comment:
Removal should be disabled for form rules that are marked as "system". They should raise: https://github.com/greenriver/hmis-warehouse/blob/26ef8c27b51dba80b065dfa9a3034722f165a15e/drivers/hmis/app/graphql/mutations/update_form_rule.rb#L19

"system" rules exist to ensure that forms that are required for application functioning -- like Project form -- do not get removed
```
actionButtons={
  <ButtonTooltipContainer
    title={rule.system ? 'System rule cannot be removed' : ''}
  >
    <IconButton
      onClick={() => deactivate({ variables: { id: rule.id } })}
      size='small'
      sx={{ height: '28px' }}
      disabled={rule.system || loading}
    >
      <DeleteIcon fontSize='small' />
    </IconButton>
  </ButtonTooltipContainer>
}
```

Reply:
Thanks for this, sorry for the regression!